index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title></title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <div id="main">
```

Comment:
I think using header would have been a lot better instead of directly going for the main section . Using header 
makes the appearance of the website a bit better and provide a better user experience(UX).